Yoast/wordpress-seo
# Introduce the use of the Yoast grunt tasks plugin

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Removing the tasks form this repository since the plugin will provide that functionality.
* Adding the task's commands to the static mappings list because JIT grunt cannot automatically detect the Yoast grunt tasks plugin.

## Test instructions

This PR can be tested by following these steps:

1. Checkout the repo.
2. Ensure you have installed all javascript packages by running `yarn`.
3. Run the following commands:
    `grunt set-version --new-version=1.1.1`
    `grunt update-version`
4. Verify that the version number has changed in the following files:
' "readme.txt","wp-seo.php","wp-seo-main.php"'

## Quality assurance


Fixes #10135
